src/python/dxpy/scripts/dx.py

```diff
@@ -1808,21 +1795,18 @@ def get(args):
     elif entity_result_class == 'app':
         get_app(entity_result, args)
     else:
-        parser.exit(3,
-                    'Error: The given object is of class ' +
-                    entity_result['describe']['class'] +
-                    ' but an object of class file, record,' +
-                    ' applet or app was expected\n')
+        err_exit('Error: The given object is of class ' + entity_result['describe']['class'] +
+                 ' but an object of class file, record,' + ' applet or app was expected', 3)
```

Comment:
Consolidate these two string literals into one now that you've unwrapped the lines.